src/styles/globals.css

```diff
@@ -53,6 +53,10 @@
 }
 
 @layer components {
+  *:disabled {
+    opacity: 0.7;
+  }
+
```

Comment:
Why this change?

Reply:
Before, the text-box wasnt visibly disabled apart from not being able to interact with it, so this makes it a bit more visible

Reply:
Did you check that this does not have any other unexpected behavior? It could have undesired effects on some text fields.

We usually do styling using TailwindCSS, inside the `class` attribute of the elements